src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    public static String Underscores() {
+        return "__________________________________________";
+    }
+
+    public static void help() {
+        String instructions = "The following is a list of commands that you can enter:\n"
+                + "1) help - For showing all commands available\n"
+                + "2) list - To display all the tasks in your list\n"
+                + "3) todo - Add a to-do task into your list (Example: todo borrow book)\n"
+                + "4) deadline - Use \" /by \" to denote deadline (Example: deadline return book /by Sunday)\n"
+                + "5) event - Use \" /at \" to denote when the event takes place (Example: event meeting /at Mon 2-4pm)\n";
```

Comment:
Perhaps the indentation for wrapped lines can be adjusted to 8 spaces more than the parent line?